### PR TITLE
feat: Coroutines 1.6.0-RC released

### PR DESCRIPTION
**Pull request by shadrina**

### CODE REVIEW

File: docs/topics/whatsnew16.md [hunk 1]
```diff
@@ -667,6 +667,20 @@ It works with all language constructs, including inline functions.
 
 Learn more about Kover on its [GitHub repository](https://github.com/Kotlin/kotlinx-kover).
 
+## Coroutines 1.6.0-RC
+
+`kotlinx.coroutines` [1.6.0-RC](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.6.0-RC) is out with 
+multiple features and improvements:
+
+* [New Kotlin/Native memory manager](#preview-of-the-new-memory-manager) support
```
Comment (p7nov) [inline docs/topics/whatsnew16.md:675]:
```suggestion
* Support for the [new Kotlin/Native memory manager](#preview-of-the-new-memory-manager)
```

more than three nouns in a row are hard to parse.

**Review by p7nov — APPROVED** [review]
👍 
Just one minor comment

File: docs/topics/whatsnew16.md (hunk 1 above)
Comment (shadrina) [reply docs/topics/whatsnew16.md:675]: Thank you!

**Review by qwwdfsad — APPROVED** [review]
